src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
This component is not exported from Design system library. You should add export to https://github.com/securityscorecard/design-system/blob/tabs-refactor/src/components/Tabs/index.ts